docs/reference/method/MongoDBClient__construct.txt

```diff
@@ -1,6 +1,6 @@
-==============================
-MongoDB\\Client::__construct()
-==============================
+=============
+__construct()
```

Comment:
I assume this was done to consolidate entries in the nav menu, but it means the page title itself is much more vague. Is there no way to preserve `MongoDB\Client::__construct()` here and only trim the navigation entry? One of my original suggestions (perhaps in Slack) was to modify the logic to generate menu content and apply some string manipulation. That would certainly require some tooling changes, but I think it's preferable to leaving the destination pages themselves without identifying titles.

As is, this page is going to show up in search results as "__construct() -- PHP Library Manual", which looks pretty bad.

Reply:
I think we need some input from the docs platform team to see what's possible here. If there's not an immediate solution available, I think reducing the page titles to just the base class name and method (e.g. `MongoDB\Client::__construct()` -> `Client::__construct()`) would be the best compromise to allow meaningful page titles.

Reply:
I reached out to docs platform and it seems like there is a way to do this! I'll change all the page titles back to the full namespace and just trim the navigation entry

Reply:
I saw the [Slack thread](https://mongodb.slack.com/archives/C018UTGP23T/p1707924824236599). Glad there's an existing solution for this, and thanks @schmalliso for the assist.